ui/src/onboarding/components/configureStep/Scraping.tsx

```diff
@@ -101,7 +103,14 @@ export class Scraping extends PureComponent<Props> {
   }
 
   private handleSelectBucket = (bucket: string) => {
-    this.props.onSetScraperTargetBucket(bucket)
+    const {buckets, onSetScraperTargetBucket, onSetBucketInfo} = this.props
+
+    const bucketID = buckets.find(b => b.name === bucket).id
+    const org = buckets.find(b => b.name === bucket).organization
+    const orgID = buckets.find(b => b.name === bucket).organizationID
```

Comment:
so rather than finding the same bucket 3 times over again, can you in one step find the bucket and then extract the additional information that you need off of it?